### PR TITLE
chore: [TECH-661] migrate List, ListItem

### PR DESCRIPTION
**Pull request by jasminenguyennn**

### CODE REVIEW

File: src/core_modules/capture-core/components/ScopeSelector/QuickSelector/Program/ProgramList.js [hunk 1]
```diff
@@ -43,29 +41,20 @@ type Props = {
 const ProgramListPlain = (props: Props) => {
     const { items, onSelect, classes } = props;
     return (
-        <List className={classes.list}>
+        <Menu className={classes.list}>
             {items.map(item =>
                 (
-                    <ListItem
+                    <MenuItem
```
Comment (jasminenguyennn) [inline src/core_modules/capture-core/components/ScopeSelector/QuickSelector/Program/ProgramList.js:47]: After: 
Screenshot 2022-12-08 at 09 34 01

File: src/core_modules/capture-core/components/Notes/Notes.component.js [hunk 2]
```diff
@@ -212,23 +209,28 @@ class NotesPlain extends React.Component<Props, State> {
         const { notes, classes, entityAccess } = this.props;
         return (
             <div className={classes.notesContainer}>
-                <List dense className={classes.notesList} data-test="comments-list">
+                <Menu dense className={classes.notesList} data-test="comments-list">
```
Comment (jasminenguyennn) [inline src/core_modules/capture-core/components/Notes/Notes.component.js:212]: ### Before
notes_before

### After
Screenshot 2022-12-08 at 09 41 33

File: src/core_modules/capture-core/components/DataEntry/dataEntryOutput/withFeedbackOutput.js [hunk 3]
```diff
@@ -50,25 +51,29 @@ const getFeedbackOutput = () =>
             (<div>
                 {feedbackItems.displayTexts &&
                     feedbackItems.displayTexts.map(item => (
-                        <ListItem
+                        <MenuItem
```
Comment (jasminenguyennn) [inline src/core_modules/capture-core/components/DataEntry/dataEntryOutput/withFeedbackOutput.js:54]: ### Before: 
Screenshot 2022-12-08 at 09 52 29


### After
Screenshot 2022-12-08 at 09 58 03

File: src/core_modules/capture-core/components/DataEntry/dataEntryOutput/withIndicatorOutput.js [hunk 4]
```diff
@@ -48,25 +50,29 @@ const getIndicatorOutput = () =>
             (<div>
                 {indicatorItems.displayTexts &&
                     indicatorItems.displayTexts.map(item => (
-                        <ListItem
+                        <MenuItem
```
Comment (jasminenguyennn) [inline src/core_modules/capture-core/components/DataEntry/dataEntryOutput/withIndicatorOutput.js:53]: Indicator should work the same as Feedback

**Review by simonadomnisoru — APPROVED** [review]

**Review by geethaalwan — APPROVED** [review]
Tested successfully on 2.40 version

**Pull request merged**